cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -183,6 +217,19 @@ pub enum Inst {
         not_taken: BranchTarget,
     },
 
+    /// A one-way conditional branch, invisible to the CFG processing; used *only* as part of
+    /// straight-line sequences in code to be emitted.
+    OneWayJmpCond { cc: CC, dst: BranchTarget },
+
+    /// Jump-table sequence, as one compound instruction (see note in lower.rs for rationale).
+    JmpTable {
```

Comment:
It might be worthwhile to go ahead and box up / out-of-line some of this, like we did with `JTSequenceInfo`, to keep the `Inst` size down.

Reply:
Added to the todo list, will probably make more sense once all the instructions have been implemented.